hmri_code_tests/hmri_test_utils.m

```diff
@@ -26,6 +26,19 @@
             % Steady state solution for dual-TR spoiled acquisition with angle in degrees
             S=hmri_test_utils.dualTRernst(deg2rad(alpha),TR1,TR2,R1);
         end
+
+        function S=HelmsMTwApprox(alpha,TR,R1,delta)
```

Comment:
I added a clarification to the documentation of this function now. In the tests using this function, a multiplicative factor is introduced so that A =/= 1, but rather some arbitrary scaling.